types/ckeditor__ckeditor5-list/src/listproperties.d.ts

```diff
@@ -1,8 +1,14 @@
 import { Plugin } from '@ckeditor/ckeditor5-core';
-import ListPropertiesEditing from './listproperties/listpropertiesediting';
-import ListPropertiesUI from './listproperties/listpropertiesui';
+import ListPropertiesEditing from './listpropertiesediting';
```

Comment:
you should follow original paths. Please see https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-list/src/listproperties
The paths should be 
```typescript
import ListPropertiesEditing from './listproperties/listpropertiesediting';
import ListPropertiesUI from './listproperties/listpropertiesui';
```

Reply:
Actually we should follow the stable release (v32), not `master`.

https://github.com/ckeditor/ckeditor5/tree/v32.0.0/packages/ckeditor5-list/src